### DISCUSSION TITLE
.Net 7 Trimming

### DISCUSSION THREAD

**Discussion by coscosname** [AvaloniaUI/Avalonia#9430] [Q&A]
How do I trim my application in .Net 7? From what I have seen the `copyused` option has disappeared :/
https://learn.microsoft.com/en-us/dotnet/core/deploying/trimming/trimming-options?pivots=dotnet-7-0

**Comment by maxkatz6** [comment]
Use [TrimmerRootDescriptor](https://learn.microsoft.com/en-us/dotnet/core/deploying/trimming/trimming-options?pivots=dotnet-7-0#root-descriptors) instead. You can find example here https://github.com/AvaloniaUI/avalonia-dotnet-templates/blob/master/templates/csharp/app/Roots.xml.
I just updated Avalonia templates, and with newer 0.10.18.2 templates version it seems to work just fine when publishing with default trim options.
